crates/witness-generator/src/eest_generator.rs

```diff
@@ -220,30 +222,137 @@ fn find_all_files_with_extension(path: &Path, extension: &str) -> Vec<PathBuf> {
 
 #[cfg(test)]
 mod tests {
+    use flate2::bufread::GzDecoder;
+    use tar::Archive;
+
     use super::*;
+    use std::{fs::File, str::FromStr};
 
-    #[test]
-    fn test_eest_generator_deletes_directory_on_drop() {
-        for delete_eest_folder in [false, true] {
-            let directory_path = PathBuf::from("./zkevm-fixtures/test-123");
-            std::fs::create_dir_all(&directory_path).unwrap();
-            assert!(directory_path.exists());
-
-            let eest_generator = ExecSpecTestBlocksAndWitnesses {
-                directory_path: directory_path.clone(),
-                include: vec![],
-                exclude: vec![],
-                delete_eest_folder,
-            };
+    fn decompress_eest_release(dest_dir: &Path) -> Result<()> {
+        let path =
+            PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("testdata/zkevm_fixtures_v0.1.0.tar.gz");
```

Comment:
Initially, I had a version where the script was called, so we always downloaded the whole release. But this had two problems: 1. GH rate limiting, 2. However, it is ultimately incorrect to base a test on having an internet connection.

Reply:
Yeah agreed, re having tests be as reproducible as possible